test-app/frontend/package.json

```diff
@@ -6,8 +6,12 @@
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
     "@general-vms/shared": "file:../shared",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.11.3",
```

Comment:
Are these needed for some reason? the `@material-ui` is for v4 which I think is deprecated or on LTS now in favor of `@mui` for v5?  We do have `@mui/icons-material` and `@mui/materual` already.

Reply:
[react-jsonschema-form](https://github.com/rjsf-team/react-jsonschema-form) say they support v5 but it throws import errors if `@material-ui` isn't installed. Can't reproduce it anymore so I will remove these packages but here's a reference for the future if this does happen again: https://github.com/rjsf-team/react-jsonschema-form/issues/2762

Reply:
Ignore the above, I was running npm install commands in the wrong dir lol. Anyway, I did find an actual fix to this, will update with that.